app/_src/mesh/features/meshglobalratelimit.md

```diff
@@ -0,0 +1,664 @@
+uses for communicating with the control plane. The control plane will periodically compute any new limits configuration and send it to ratelimit service.
+
+### Rate limiting algorithm
+
+The ratelimit service uses a fixed window algorithm. It allocates a new counter for each time unit. Let's assume that we have configured limits to 
+10 requests per minute. At the beginning of each minute, ratelimit service will create a new counter. 
+
+<center>
+  <img src="/assets/images/docs/mesh/ratelimit-algorithm.png"/>
+</center>
+> Figure 2: Diagram showing how rate limit counters are placed on timeline and how request is qualified for specific time window.
+
+When a new request arrives, it updates the counters in the window based on the request timestamp. Be aware of the following when configuring counters:
+
+* The counter can be depleted at the beginning of the window.
+* The first counter can be depleted at the end of the window and the second can be depleted at the beginning of the window.
+
+The first scenario can be problematic for long time units like hours and days. Because the limit is reached at the beginning of the time window,
+your service will not serve any requests for the rest of the time window. To solve this issue, you can split your limit from 60 per hour to 
+one per minute.
+
+The second scenario could result in requests burst around the window switch. In this scenario, when you configure the ratelimit to 100 requests per minute
+you could end up receiving 200 request in a couple of seconds.
+
+### Multizone deployment
+
+When it comes to multizone deployment, you should deploy the ratelimit service in every zone. As for Redis, you have two options:
+
+<center>
+  <img src="/assets/images/docs/mesh/ratelimit-service-multizone-multi-redis.png"/>
+</center>
+> Figure 3: Diagram of multizone ratelimit setup with Redis per zone.
+
+The first option is to deploy Redis in every zone. In this setup, limits will be applied per zone. Since each zone will have its own counters cache, 
+requests will be faster, and it will be easier to distribute your system geographically.
+
+<center>
+  <img src="/assets/images/docs/mesh/ratelimit-service-multizone-single-redis.png"/>
+</center>
+> Figure 4: Diagram of multizone ratelimit setup with global Redis.
+
+The second option is to deploy a single Redis datastore for all your zones. In this setup, the rate limit will be truly global. When deploying a single Redis datastore, remember that if your zones are distributed geographically requests to Redis can become slower which could drastically 
+increase response times of service you are rate limiting. 
+
+### Securing communication between ratelimit service and Control Plane
+
+Communication between the ratelimit service and control plane is encrypted. In addition, zone token authorization is used on Universal and  service account authorization is used on Kubernetes. 
```

Comment:
I thought this wasn't done yet

Reply:
It is not done yet, but I've added this part hoping Bart will finish this, I can comment out this section for now and we will add it back when it is done

Reply:
I commented it out for now